src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -38,13 +38,13 @@
 
         public static bool RaiseUnhandledExceptionInDriverCode { get; set; }
 
-        protected virtual Tool Tool { get; set; }
+        public virtual Tool Tool { get; set; }
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

This property needs to be public in order to interact with it properly in scenarios where users proactively create a SARIF logger (and where skimmers may be loaded as extensions).